.github/docker_images/cmake_build_versions/awslc_build.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0 OR ISC
+
+set -ex -o pipefail
+
+echo "Building with CMake Version: $(cmake --version)"
+
+BUILD_DIR=$(mktemp -d)
+SRC_DIR="${SRC_DIR:-/awslc}"
+CMAKE_CONFIGURE_ARGS=("${argv[@]}")
+
+pushd "${BUILD_DIR}"
+
+cmake "${SRC_DIR}" "${CMAKE_CONFIGURE_ARGS[@]}"
+cmake --build .
+
+"${BUILD_DIR}/crypto/crypto_test"
+
+if [[ -a "${BUILD_DIR}/ssl/ssl_test"  ]]; then
```

Comment:
when would we not build libssl?

Reply:
By default we do, but since I have the script taking in the CMake configuration arguments in a generic manner it could be disabled and not compiled in the build if we wanted to. But at the moment in this PR, it will always be built and thus always tested.